stable/mongodb/0.4.7/chart/values.yaml

```diff
@@ -0,0 +1,74 @@
+_config:
+  _metadata:
+    name: "mongodb"
+    version: "0.4.7"
+    description: ""
+  mongodb:
+    containers:
+      mongodb:
+        image: "bitnami/mongodb:3.4.2-r0"
```

Comment:
How do you make sure modifying the image will work?  what if user changes to mongodb:2.6, or even mysql :)

Validation is needed, but that's per chart and that's certainly amount of work we can't afford.  e.g. you need to check if mongodb 3.2 is compatible with 3.3 feature-wise.

In short, modifying image is dangerous and most of user just want it to work out of box.